recipes-core/packagegroups/packagegroup-ni-base.bb

```diff
@@ -17,7 +17,6 @@ MACHINE_ESSENTIAL_EXTRA_RRECOMMENDS ?= ""
 
 ALL_DISTRO_ARM_PACKAGES = "\
```

Comment:
Why aren't you removing all ARM-specific packages entirely? I have a few guesses, but want to hear from you explicitly.

Reply:
I'm going to do that in an independent patchset, once I have everything in the base packagegroup building and the concerning warnings are resolved.